opslevel/datasource_opslevel_scorecards.go

```diff
@@ -0,0 +1,105 @@
+package opslevel
+
+import (
+	"github.com/hashicorp/terraform-plugin-sdk/helper/schema"
+	"github.com/opslevel/opslevel-go/v2023"
+)
+
+func datasourceScorecards() *schema.Resource {
+	return &schema.Resource{
+		Read: wrap(datasourceScorecardsRead),
+		Schema: map[string]*schema.Schema{
+			"ids": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"names": {
+				Type:     schema.TypeList,
+				Computed: true,
+				Elem:     &schema.Schema{Type: schema.TypeString},
+			},
+			"owner_id": {
```

Comment:
i don't think this field name matches what your a doing in `d.Set("owner_ids"`

The schema is there to define what fields the customer can use (and it help generate the documentation) but it has to match all the `d.Set` or `d.Get` calls